web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -41,10 +41,32 @@ import AgentButtonAdd from 'teleport/components/AgentButtonAdd';
 import { SearchResource } from 'teleport/Discover/SelectResource';
 import { encodeUrlQueryParams } from 'teleport/components/hooks/useUrlFiltering';
 import Empty, { EmptyStateInfo } from 'teleport/components/Empty';
+import { FeatureFlags } from 'teleport/types';
 
 import { ResourceActionButton } from './ResourceActionButton';
 import SearchPanel from './SearchPanel';
 
+const getAvailableKinds = (flags: FeatureFlags) => {
+  const kinds = [];
+  if (flags.nodes) {
+    kinds.push('node');
+  }
+  if (flags.databases) {
+    kinds.push('db');
+  }
+  if (flags.desktops) {
+    kinds.push('windows_desktop');
+  }
+  if (flags.kubernetes) {
+    kinds.push('kube_cluster');
+  }
+  if (flags.applications) {
+    kinds.push('app');
+  }
+
+  return kinds;
+};
+
```

Comment:
This seems ugly (but at least its explicit). Happier to a cleaner way if its just as easy to read

Reply:
don't think it's ugly, it's simple and easy to read at least :)